templates/shared_services/gitea/terraform/gitea-webapp.tf

```diff
@@ -10,8 +18,15 @@ resource "azurerm_app_service" "gitea" {
     "WEBSITES_PORT"                  = "3000"
     "WEBSITE_VNET_ROUTE_ALL"         = 1
 
-    TRE_ID            = var.tre_id
-    RESOURCE_LOCATION = var.location
+     # Settings for private Container Registires  
+    "DOCKER_REGISTRY_SERVER_USERNAME"            = var.docker_registry_username
+    "DOCKER_REGISTRY_SERVER_URL"                 = "https://${var.docker_registry_server}"
+    "DOCKER_REGISTRY_SERVER_PASSWORD"            = var.docker_registry_password
+   
+    # TBD, Username and password should not be defined here: #542
+    GITEA_USERNAME                 = "giteaadmin"
+    GITEA_PASSWD                   = random_password.gitea_passwd.result
```

Comment:
Can we save the password in keyvault, as per https://github.com/microsoft/AzureTRE/blob/b80eb68503e5b2f4f5368f0a36e247ec6b3ffb6b/templates/core/terraform/resource_processor/vmss_porter/main.tf#L40

Reply:
@marrobi Sure, we are working on it as part of a different task (#542 ) just in order to make it in a separate PR